src/Tiamat_oxDNA.py

```diff
@@ -241,8 +241,8 @@ def getBaseConfig(base):
     else:
         # single stranded case, not really treated (could randomize orientations?)
         cm_pos = base_vector
-        a3_vector = [0, 0, 1]
-        a1_vector = [0, 1, 0]
+        a3_vector = np.array([0, 0, 1])
```

Comment:
I don't think this is going to be an issue, but could you initialise the arrays so that their dtype is float? Something like `np.array([0., 0., 1.])`